config/splat.us.stmad.yaml

```diff
@@ -62,7 +62,9 @@ segments:
       - [0xD874, rodata]
       - [0xD884, rodata]
       - [0xD8C8, c]
-      - [0xE964, c]
+      - [0xE964, c, ../random]
+      - [0xE994, c, ../update]
+      - [0xEC90, c, CollectGold]
```

Comment:
Hmm if the 3rd function is not UpdateStageEntities that weakens my theory about how the overlays are structured. MAD is sort of a special case though. I think either they just copy-pasted or did something like:
```
stXXX:
  stXXX.c shared1.c shared2.c shared3.c etc.
```